kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -401,6 +406,35 @@ if [ "$WINDOWS_CUSTOM_PACKAGE" != "" ]; then
     log_level -i "Done updating Windows custom package URL details."
 fi
 
+#####################################################################################
+#custom vnet config 
+if [ "$CUSTOM_VNET_NAME" != "" ]; then
+    log_level -i "Setting general custom vnet properties."
+    MASTER_VNET_ID="/subscriptions/$TENANT_SUBSCRIPTION_ID/resourceGroups/$RESOURCE_GROUP_NAME/providers/Microsoft.Network/virtualNetworks/$CUSTOM_VNET_NAME/subnets/$MASTER_SUBNET_NAME"
+    AGENT_VNET_ID="/subscriptions/$TENANT_SUBSCRIPTION_ID/resourceGroups/$RESOURCE_GROUP_NAME/providers/Microsoft.Network/virtualNetworks/$CUSTOM_VNET_NAME/subnets/$AGENT_SUBNET_NAME"
+
+    cat $AZURESTACK_CONFIGURATION | \
```

Comment:
Is it possible to have only windows nodes in the cluster? If yes, we need to add the condition "if [ "$LINUX_AGENT_COUNT" != "0" ]" condition

Reply:
good point. Let handle that situation as well.